### PR TITLE
rint intrinsics: caution against actually trying to check for floating-point exceptions

### PR DESCRIPTION
**Pull request by RalfJung**

### CODE REVIEW

File: library/core/src/intrinsics.rs [hunk 1]
```diff
@@ -1787,6 +1787,8 @@ extern "rust-intrinsic" {
     /// so this rounds half-way cases to the number with an even least significant digit.
     ///
     /// May raise an inexact floating-point exception if the argument is not an integer.
+    /// However, Rust assumes floating-point exceptions cannot be bserved, so this is not something that
```
Comment (cuviper) [inline library/core/src/intrinsics.rs:1790]: I assume this is a typo? (on both additions)
```suggestion
    /// However, Rust assumes floating-point exceptions cannot be observed, so this is not something that
```

Comment (RalfJung) [reply library/core/src/intrinsics.rs]: *oops* fixed

File: library/core/src/intrinsics.rs [hunk 2]
```diff
@@ -1787,6 +1787,8 @@ extern "rust-intrinsic" {
     /// so this rounds half-way cases to the number with an even least significant digit.
     ///
     /// May raise an inexact floating-point exception if the argument is not an integer.
+    /// However, Rust assumes floating-point exceptions cannot be observed, so this is not something that
```
Comment (nagisa) [inline library/core/src/intrinsics.rs:1790]: Somewhat ambiguous what "this" in "this is not something that can actually be used" is. Is it this intrinsic itself? Or the very fact that the exceptions might be raised and you cannot utilize this fact for anything.

Comment (cuviper) [reply library/core/src/intrinsics.rs:1790]: It was clear to me that "this" meant the raised exception, but we can always tweak it further!

**Pull request merged**